dask_planner/src/sql/table.rs

```diff
@@ -36,6 +37,36 @@ impl TableSource for DaskTableSource {
     fn schema(&self) -> SchemaRef {
         self.schema.clone()
     }
+
+    // temporarily disable clippy until TODO comment below is addressed
+    #[allow(clippy::if_same_then_else)]
+    fn supports_filter_pushdown(
+        &self,
+        filter: &Expr,
+    ) -> datafusion_common::Result<TableProviderFilterPushDown> {
+        let mut filters = vec![];
+        split_conjunction(filter, &mut filters);
+        if filters.iter().all(|f| is_supported_push_down_expr(*f)) {
+            // TODO this should return Exact but we cannot make that change until we
+            // are actually pushing the TableScan filters down to the reader because
+            // returning Exact here would remove the Filter from the plan
```

Comment:
Is this TODO an action item on the dask-sql side?

Reply:
Yes. With this change, the logical plan will now have filters pushed down to the `TableScan` and those need mapping over to the actual read operation.

Reply:
https://github.com/dask-contrib/dask-sql/issues/596 made an issue for this FYI